app/policies/supervisor_policy.rb

```diff
@@ -1,2 +1,5 @@
 class SupervisorPolicy < UserPolicy
+  def index?
+    user&.casa_admin? || user&.supervisor?
```

Comment:
Why do we need the safe navigator? Do we use Policies for non authenticated users?

Reply:
This was originally in @drborges commit that I cherry-picked. I'll  let him  speak to that.

Reply:
This is more like a temporary solution because the sidebar currently shows up on the login page. @jasperfurniss do we have a fix for that yet? I can work on it if not and clean this up.

Reply:
nope, nvm, I just verified that's no longer the case... @jasperfurniss can you pull the latest changes from `new-layout` and remove the safe-navigator from here, please? I shipped my PR a few minutes ago 😬

Reply:
Yes @drborges I've got some merge conflicts.. and was just about to do that actually.

Reply:
@drborges Yeah @ds29  and I fixed the sidebar from showing up on login page earlier today.